config.example.toml

```diff
@@ -33,6 +34,12 @@ usermap = "fedora_jira_usermap.toml"  # External file, can be reused
 # fedora_user1 = "jira_user1"
 # fedora_user2 = "jira_user2"
 
+# This section converts specific label on repository to story point
+[instances."pagure.io".story_points]
```

Comment:
To me, using the same name for concrete story points of an issue (an integer) and this mapping is confusing, maybe call this one `labels_to_story_points`? See e.g. the changes in `jira_sync/repositories/base.py`.